docs/PROVISIONING_AWS.md

```diff
@@ -0,0 +1,102 @@
+
+In addition to _cloning this repo_, you'll need the following:
+
+* Access to an AWS account with the proper policies to create resources (see details below)
+* Docker installed
+  * RHEL/CentOS: `yum install -y docker`
+  * Fedora: `dnf install -y docker`
+  * **NOTE:** If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+
+```
+cd ~/src/
+git clone https://github.com/redhat-cop/casl-ansible.git
+```
+
+* Download/untar `openshift-ansible` for use as part of the install. Make sure to do so within the same directory as above (i.e.: `~/src`), and either rename the directory to `openshift-ansible` (or create symlink to it - see example below). See our [Compatibility Matrix](./README.md#compatability-matrix) for versions / urls to be used for the download.
+
+```
+cd ~/src/
+wget <url> -O - | tar -xz
+ln -fs openshift-ansible-*<version>* openshift-ansible
+```
+
+* Run `ansible-galaxy` to pull in the necessary requirements for the CASL provisioning of OpenShift on AWS:
+
+```
+cd ~/src/casl-ansible
+ansible-galaxy install -r casl-requirements.yml -p roles
+```
+
+## AWS specific requirements
+* Available parameters to use the AWS provision can be found in the Role's [README](roles/manage-aws-infra/README.md)
+* A [Key-pair in AWS](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair)
+* Modify 'regions' entry (line 13) in the inventory 'ec2.ini' file to match the 'aws_region' variable in your inventory
+* Modify 'instance_filters' entry (line 14) in the inventory 'ec2.ini' file to match the 'env_id' variable in your inventory's `all.yml`
+
+Cool! Now you're ready to provision OpenShift clusters on AWS
+
+## Provision an OpenShift Cluster
+
+As an example, we'll provision the `sample.aws.example.com` cluster defined in the `~/src/casl-ansible/inventory` directory.
+
+> **Note**: *It is recommended that you use a different inventory similar to the ones found in the `~src/casl-ansible/inventory` directory and keep it elsewhere. This allows you to update/remove/change your casl-ansble source directory without losing your inventory. Also note that it may take some effort to get the inventory just right, hence it is very beneficial to keep it around for future use without having to redo everything.*
+
+The following is just an example on how the `sample.aws.example.com` inventory can be used:
+
+1) Edit `~/src/casl-ansible/inventory/sample.aws.example.com.d/inventory/group_vars/all.yml` to match your AWS environment. See comments in the file for more detailed information on how to fill these in.
+
+2) Edit `~/src/casl-ansible/inventory/sample.aws.example.com.d/inventory/group_vars/OSEv3.yml` for your AWS specific configuration. See comments in the file for more detailed information on how to fill these in.
+
+3) Edit `~/src/casl-ansible/inventory/sample.aws.example.com.d/inventory/ec2.ini` to update `regions` and `instance_filters` as described above.
```

Comment:
Note that point 3 is no longer required once we merge the aws-gluster PR as now some inventory is generated dynamically from inventory variables

Reply:
@makentenza yeah, saw that in your PR. Let's keep this as-is and update it in your PR.